dodo.py

```diff
@@ -598,12 +662,77 @@ def deploy_to_prod():
                 f'{curl}',
                 f'echo "{describe}"',
                 f'{describe}',
-            ] if deploy_to_prod() else [
-                f'attempting to deploy to prod without env var DEPLOY_TO=prod set',
-                'false',
             ],
         }
 
+@guard('prod')
+def task_deploy_svc():
+    '''
+    run serverless deploy -v on a provided <svc>
+    '''
+    servicepath = f'services/%(svc)s'
+    curl = f'curl --silent https://{CFG.DEPLOYED_ENV}.%(svc)s.mozilla-subhub.app/v1/version'
```

Comment:
URL needs to by updated by service name.

Reply:
it is updated by the service name; confused

Reply:
URLs are now in the form of `https://{CFG.DEPLOYED_ENV}.%(svc)s.mozilla-subhub.app/v1/{sub/hub}/version`